src/libraries/System.Net.Http/src/System/Net/Http/Headers/QPackStaticTable.cs

```diff
@@ -96,7 +96,7 @@ internal static class QPackStaticTable
             (new HeaderDescriptor(KnownHeaders.Authorization), ""), // 84
             (new HeaderDescriptor(KnownHeaders.ContentSecurityPolicy), "script-src 'none'; object-src 'none'; base-uri 'none'"), // 85
             (new HeaderDescriptor("early-data"), "1"), // 86
-            (new HeaderDescriptor("expect-ct"), ""), // 87
```

Comment:
This is an existing bug in HTTP/3 where `x-xss-protection` and `expect-ct` lookups/removes would silently fail if added via `OnStaticIndexedHeader`.